indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -0,0 +1,33 @@
+(ns cmr.indexer.data.concepts.service
+  "Contains functions to parse and convert service and service association concepts."
+  (:require
+   [cmr.common.log :refer (debug info warn error)]
+   [cmr.indexer.data.concept-parser :as concept-parser]
+   [cmr.transmit.metadata-db :as mdb]))
+
+(defn- service-association->service-concept
+  "Returns the service concept and service association for the given service association."
+  [context service-association]
+  (let [{:keys [service-concept-id]} service-association
+        service-concept (mdb/find-latest-concept
+                         context
+                         {:concept-id service-concept-id
+                          :latest true}
+                         :service)]
+    (when-not (:deleted service-concept)
+      service-concept)))
+
+(defn- has-formats?
+  "Returns true if the given service has more than one SupportedFormats value."
+  [context service-concept]
+  (let [service (concept-parser/parse-concept context service-concept)
+        supported-formats (get-in service [:ServiceOptions :SupportedFormats])]
+    (> (count supported-formats) 1)))
```

Comment:
is it possible for supported-formats to not be a collection? If it is a string or something this will pass even if there is only 1 returned.

Reply:
UMM-S schema has SupportedForms as a list of enums, so no. It will always be a list.